pages/common/peerflix.md

```diff
@@ -0,0 +1,15 @@
+# peerflix
+
+> Stream torrents to your favorite player.
+
+- Stream a torrent:
+
+`peerflix "magnet:?xt=urn:btih:ef330b39f4801d25b4245212e75a38634bfc856e"`
```

Comment:
IMO it would be nice to include a note in the description for this example saying what this magnet link refers to.

Reply:
Can we just dump a random string here ? Pointing to a real magnet url does not seem very elegant to me.

I want this to be similar to examples of urls like - `www.example.com`.

Reply:
That works too. Preferably something obviously example-like, such as `000011112222...eeeeffff`, or something cleverer inspired by https://en.wikipedia.org/wiki/Hexspeak#Notable_magic_numbers or http://bada55.io/